transformer/transformer.go

```diff
@@ -580,6 +581,11 @@ func runSingleTransform(artifactsToProcess, allArtifacts []transformertypes.Arti
 				"pathMappings":      summarizePathMappings(newPathMappings),
 			},
 		)
+		// transformers that are invoked by default has source vertex as start
+		if tconfig.Spec.InvokedByDefault.Enabled {
+			edgeName := fmt.Sprintf("%d -> %d (invoked by default)", 0, targetVertexId)
+			graph.AddEdge(0, targetVertexId, edgeName, nil)
```

Comment:
Instead of hard coding `0`  as the start vertex Id, we can make it a constant that is shared between all the packages that need it.